src/components/EventRow.jsx

```diff
@@ -0,0 +1,16 @@
+import React, { PropTypes, Component } from 'react'
+
+export default class EventRow extends Component {
+  static propTypes = {
+    param: PropTypes.string.isRequired
```

Comment:
this class is a placeholder that I hadn't worked on at all yet :).  It's fleshed out in another branch.  I implemented displaying events for week/day and month in separate branches to make pull requests simpler.  In this case, I would do the pull request to the streamline branch (this one) so you can see the difference more clearly.